.testing/tools/parse_fms_clocks.py

```diff
@@ -60,18 +60,34 @@ def main():
         print(json.dumps(config))
 
 
-def parse_mom6_param(param_file):
+def parse_mom6_param(param_file, header=None):
     params = {}
     for line in param_file:
         param_stmt = line.split('!')[0].strip()
-        if param_stmt:
-            key, val = [s.strip() for s in param_stmt.split('=')]
-
-            # TODO: Convert to equivalent Python types
-            if val in ('True', 'False'):
-                params[key] = bool(val)
-            else:
-                params[key] = val
+
+        # Skip blank lines
+        if not param_stmt:
+            continue
```

Comment:
`continue` -> `pass`

Reply:
`pass` is a null operation, whereas `continue` terminates the loop iteration loop and begins the next one.  Using `pass` would leave `value` undefined (or worse, defined from the previous loop) and have unexpected effects.